klippa/src/serialize.rs

```diff
@@ -165,6 +178,12 @@ impl Serializer {
         !!self.errors
     }
 
+    pub(crate) fn only_overflow(&self) -> bool {
+        self.errors == SerializeErrorFlags::SERIALIZE_ERROR_OFFSET_OVERFLOW
+            || self.errors == SerializeErrorFlags::SERIALIZE_ERROR_INT_OVERFLOW
+            || self.errors == SerializeErrorFlags::SERIALIZE_ERROR_ARRAY_OVERFLOW
+    }
```

Comment:
iirc this check is returning `false` if the bitset contains some combination of these flags?

Reply:
Good question! It's ported from Harfbuzz. I think these flags are exclusive, maybe @garretrieger can help to confirm?

Reply:
Not sure why specifically it's that way in hb but unless we have a strong reason otherwise we should just match hb.